includes/admin/forms/class-metabox-form-data.php

```diff
@@ -836,18 +836,29 @@ public function save( $post_id, $post ) {
 
 								// Arrange repeater field keys in order.
 								$form_meta_value = array_values( $form_meta_value );
-
-								// Save data.
-								give_update_meta( $post_id, $form_meta_key, $form_meta_value );
 								break;
 
 							default:
 								$form_meta_value = give_clean( $_POST[ $form_meta_key ] );
-
-								// Save data.
-								give_update_meta( $post_id, $form_meta_key, $form_meta_value );
 						}
 
+
+						/**
+						 * Filter the form meta value before saving
+						 *
+						 * @since 1.8.9
+						 */
+						$form_meta_value = apply_filters(
```

Comment:
@DevinWalker I saw some form meta value:
1.`_give_set_value`
2. `_give_custom_amount_minimum`
3. `_give_donation_levels[$index][_give_amount]`

storing with currency formatting which is not good. Amount should store only with decimal formatting.